apps/api/nest-cli.json

```diff
@@ -18,14 +18,6 @@
       {
         "include": ".env.production",
         "outDir": "dist/src"
-      },
-      {
-        "include": "app/layouts/usecases/get-novu-layout/templates/layout.handlebars",
-        "outDir": "dist/src"
-      },
-      {
-        "include": "app/content-templates/usecases/compile-email-template/templates/basic.handlebars",
-        "outDir": "dist/src"
```

Comment:
this configuration to copy handlebars files is moved to the `application-generic` package, we have there the `build:copy-template` npm script